nilearn/datasets/struct.py

```diff
@@ -166,21 +329,23 @@ def fetch_icbm152_brain_gm_mask(data_dir=None, threshold=0.2, resume=True,
 
     threshold : float, optional
         The parameter which amounts to include the values in the mask image.
-        The values lies above than this threshold will be included. Defaults
-        to 0.2 (one fifth) of values.
-        Default=0.2.
+        Values lying above this threshold will be included. Default=0.2
 
     resume : bool, optional
         If True, try resuming partially downloaded data.
         Default=True.
 
+    n_iter: int, optional
```

Comment:
Same, if `n_iter` is a new parameter, I would add a versionadded sphinx directive for it.